azurerm/resource_arm_log_analytics_workspace_linked_service.go

```diff
@@ -63,10 +63,13 @@ As such the existing 'azurerm_log_analytics_workspace_linked_service' resource i
 			},
 
 			"linked_service_properties": {
-				Type:     schema.TypeList,
-				Optional: true,
-				Computed: true,
-				ForceNew: true,
+				Type:          schema.TypeList,
+				Optional:      true,
+				Computed:      true,
+				ForceNew:      true,
+				ConflictsWith: []string{"resource_id"},
+				MaxItems:      1,
+				Deprecated:    "This property has been deprecated in favour of the 'resource_id' property and will be removed in version 2.0 of the provider",
```

Comment:
can we also add this [to the 2.0 upgrade guide](https://github.com/terraform-providers/terraform-provider-azurerm)?